lib/activerecordbase.rb

```diff
@@ -66,8 +66,8 @@ def queue_create(attributes_of_interest)
   def queue_update(attributes_of_interest)
     transaction_changed_attributes.each do |k, v|
       next unless attributes_of_interest.include?(k.to_sym)
-      # new_val = (self.class.type_for_attribute(k).type == :datetime ? v.in_time_zone(BqStream.timezone) : v).to_s
-      BqStream::QueuedItem.create(table_name: self.class.to_s, record_id: id, attr: k, new_value: v.to_s)
+      new_val = (self.class.type_for_attribute(k).type == :datetime ? v.in_time_zone(BqStream.timezone) : v).to_s
+      BqStream::QueuedItem.create(table_name: self.class.to_s, record_id: id, attr: k, new_value: new_val)
```

Comment:
extract the time zone conversion into a method to make this a little DRYer and easier to maintain.